deploy_exchange_1_0/action.yml

```
@@ -0,0 +1,33 @@
+name: Deploy to Exchange 1.0
```

There is no concept Exchange 1.0 or 2.0 (different with CloudHub) so we can remove it 👌 

```suggestion
name: Deploy to Exchange
```

---

Solved in https://github.com/nimblehq/mulesoft-actions/pull/12/commits/3c4989ac6804bd8802099e792d2470cda98d4544